samples/powerplay/src/main/kotlin/com/example/powerplay/MainActivity.kt

```diff
@@ -0,0 +1,529 @@
+                        text = playList[it].name, fontSize = 24.sp,
+                        color = Color.Black,
+                        style = TextStyle(fontWeight = FontWeight.ExtraBold)
+                    )
+                }
+                Spacer(modifier = Modifier.height(8.dp))
+                AnimatedContent(targetState = playingSongIndex.intValue, transitionSpec = {
+                    (scaleIn() + fadeIn()) with (scaleOut() + fadeOut())
+                }, label = "") {
+                    Text(
+                        text = playList[it].artist, fontSize = 12.sp, color = Color.Black,
+                        style = TextStyle(fontWeight = FontWeight.Bold)
+                    )
+                }
+
+                Spacer(modifier = Modifier.height(16.dp))
+
+
+                /***
+                 * Includes animated song album cover
+                 */
+                HorizontalPager(
+                    modifier = Modifier.fillMaxWidth(),
+                    state = pagerState,
+                    pageSize = PageSize.Fixed((configuration.screenWidthDp / (1.7)).dp),
+                    contentPadding = PaddingValues(horizontal = 85.dp)
+                ) { page ->
+
+                    val painter = painterResource(id = playList[page].cover)
+
+                    if (page == pagerState.currentPage) {
+                        VinylAlbumCoverAnimation(isSongPlaying = isPlaying.value, painter = painter)
+                    } else {
+                        VinylAlbumCoverAnimation(isSongPlaying = false, painter = painter)
+                    }
+                }
+                Spacer(modifier = Modifier.height(54.dp))
+                Column(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(horizontal = 32.dp),
+                ) {
+                }
+                Spacer(modifier = Modifier.height(8.dp))
+                Text(
+                    "Performance Modes"
+                )
+                Spacer(modifier = Modifier.height(8.dp))
+                Column {
+                    val radioOptions = mutableListOf("None", "Low Latency", "Power Saving")
+                    if (isOffloadSupported) radioOptions.add("PCM Offload")
+
+                    val (selectedOption, onOptionSelected) = remember {
+                        mutableStateOf(radioOptions[0])
+                    }
+                    radioOptions.forEachIndexed { index, text ->
+                        Row(
+                            Modifier
+                                .height(32.dp)
+                                .selectable(
+                                    selected = (text == selectedOption),
+                                    onClick = {
+                                        onOptionSelected(text)
+                                        offload.intValue = index
+                                        player.teardownAudioStream()
```

Comment:
The UI freezes for a second or two when switching from a PCM offload stream to another stream type

Reply:
Same a above.